templates/help.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>CLV Predictor</title>
+    <link rel="stylesheet" href="/static/styles.css" />
+  </head>
+
+  <body>
+    <header>
+      <img src="/static/fintech-transparent.png" alt="Your Logo" id="logo" />
+      <div class="buttons">
+        <button id="feedbackBtn">Feedback</button>
+        <button id="helpBtn">Help?</button>
+      </div>
+    </header>
+
+    <h1 class="title">Customer Lifetime Value Predictor.</h1>
+    <div class="flex-container">
+      <div class="column">
+        <section class="how-to-use">
+          <h2>How To Use?</h2>
+          <ol>
+            <li>
+              Enter the number of upcoming months that you want the valuation to
+              cover.
+            </li>
+            <li>
+              Select a CSV file (drag and drop into the box) or click on the
+              drag and drop box to choose a file from your device.
+            </li>
+            <li>
+              Wait for the file to be processed and the results to be displayed.
+            </li>
+          </ol>
+          <p>
+            Note: The file MUST be in .csv format and follow the format provided
```

Comment:
```suggestion
            <b>Note:</b> The file MUST be in .csv format and follow the format provided
```

Perhaps can bold out `Note:`?